packages/vuetify/src/components/VForm/VForm.ts

```diff
@@ -131,7 +131,7 @@ export default mixins(
         ...this.attrs$,
       },
       on: {
-        submit: (e: Event) => this.$emit('submit', e),
+        ...this.listeners$,
```

Comment:
seems like destructing is not necessary
![image](https://user-images.githubusercontent.com/15625235/78544546-972c1780-77fa-11ea-83b7-eea58abc44a1.png)

Reply:
Done!